tests/copilot_helper/test_command_copilot.py

```diff
@@ -478,16 +480,17 @@ def test_exit_if_no_local_copilot_environments(self, fakefs):
     )
     @patch("dbt_copilot_helper.jinja2_tags.version", new=Mock(return_value="v0.1-TEST"))
     def test_addons_parameters_file_included_with_required_parameters_for_the_addon_types(
-        self, fakefs, addon_file_contents, addon_type
+        self, fakefs, addon_file_contents, has_postgres_addon
     ):
         create_test_manifests(addon_file_contents, fakefs)
 
         result = CliRunner().invoke(copilot, ["make-addons"])
 
         assert result.exit_code == 0
-        assert (
-            "File copilot/environments/addons/addons.parameters.yml created" in result.output
-        ), f"addons.parameters.yml should be included for {addon_type}"
+        if has_postgres_addon:
+            assert (
+                "File copilot/environments/addons/addons.parameters.yml created" in result.output
+            ), f"addons.parameters.yml should be included for Postgres addons"
```

Comment:
This assertion (and the ones below) could perhaps be written: assert ("File...created" in result.output == has_postgres_addon, "addons...")

Then it will be testing the presence of the notification (or content in the ones below) if there's a postgres addon, and the absence if not.

Reply:
Actually, that if above should not be there, commit incoming for removing that one and kind of taking the other in via a helper method to keep the test output clearer.